include/rem_dtmf.h

```diff
@@ -0,0 +1,21 @@
+/**
+ * @file rem_dtmf.h  DTMF Decoder
+ *
+ * Copyright (C) 2010 Creytiv.com
+ */
+
+struct dtmf_dec;
+
+/**
+ * Defines the DTMF decode handler
+ *
+ * @param digit Decoded DTMF digit
+ * @param arg   Handler argument
+ */
+typedef void (dtmf_dec_h)(char digit, void *arg);
```

Comment:
the callback API does not support detecting the "end" of a digit, or the duration of a digit.

is this useful to add now ?

Reply:
IMO this is not needed. End can be communicated as digit=0, if ever needed in the future within the current API.